GameServer/Logic/Content/RuneCollection.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LeagueSandbox.GameServer.Logic.Content
+{
+    public class RuneCollection
+    {
+        private Dictionary<int, int> _runes;
+
+        public RuneCollection()
+        {
+            _runes = new Dictionary<int, int>();
+        }
+
+        public Dictionary<int, int> GetRunes()
```

Comment:
This is the same as exposing the `_runes` dictionary. Either return a copy of it, thus breaking the reference, or just use a C# getter/setter